### PR TITLE
CD->CID changes

### PR DESCRIPTION
**Pull request by iherman**
References to Controller Document have been changed to Controlled Identifier Document.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/19.html" title="Last updated on Dec 20, 2024, 9:58 AM UTC (2bb98ba)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/19/ae5158d...2bb98ba.html" title="Last updated on Dec 20, 2024, 9:58 AM UTC (2bb98ba)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -888,7 +892,7 @@ <h4>Cryptosuites</h4>
 
 
           <p class="note">
-            A common characteristics of all these cryptosuites is that keys must always be encoded using the <a data-cite="CONTROLLER-DOCUMENT#multikey">Multikey</a> encoding. 
+            A common characteristics of all these cryptosuites is that keys must always be encoded using the <a data-cite="CONTROLLER-DOCUMENT#multikey">Multikey</a> encoding.
```
Comment (TallTed) [inline index.html:895]:
```suggestion
            A common characteristic of all these cryptosuites is that keys must always be encoded using the <a data-cite="CONTROLLER-DOCUMENT#multikey">Multikey</a> encoding.
```

File: index.html [hunk 2]
```diff
@@ -1117,7 +1121,7 @@ <h3>Working Group Notes</h3>
         <dl>
           <dt><a href="https://www.w3.org/TR/vc-use-cases/">Verifiable Credentials Use Cases</a></dt>
           <dd>
-            The use cases outlined in this documents were, and are, instrumental in making progress toward standardization and interoperability of both low– and high–stakes claims with the goals of storing, transmitting, and receiving digitally verifiable proof of attributes such as qualifications and achievements. 
+            The use cases outlined in this document were, and are, instrumental in making progress toward standardization and interoperability of both low– and high–stakes claims with the goals of storing, transmitting, and receiving digitally verifiable proof of attributes such as qualifications and achievements.
```
Comment (TallTed) [inline index.html:1124]:
```suggestion
            The use cases outlined in this document were, and are, instrumental in making progress toward standardization and interoperation of both low– and high–stake claims, with the goals of storing, transmitting, and receiving digitally verifiable proof of attributes such as qualifications and achievements.
```

File: index.html [hunk 3]
```diff
@@ -1151,7 +1155,7 @@ <h3>Standard Vocabularies</h3>
 
           <dt><a href="https://w3id.org/security">Security Vocabulary</a></dt>
           <dd>
-            Definition of property names defined by the <a href="#s_di">VC Data Integrity</a> specification or the Controller Document.
+            Definition of property names defined by the <a href="#s_di">VC Data Integrity</a> specification or the Controlled Identifier Document.
```
Comment (TallTed) [inline index.html:1158]: It seems like there must be a better way to phrase this, than `Definition of property names defined by`. Does this work?
```suggestion
            Property name definitions in the <a href="#s_di">VC Data Integrity</a> specification or the Controlled Identifier Document.
```

**Comment by iherman** [comment]
The comments of @TallTed came after the PR was merged; they are raised as a separate PR in https://github.com/w3c/vc-overview/pull/20.

**Pull request merged**